### PR TITLE
navigation added

### PR DESCRIPTION
**Pull request by Ulbossya**
navigation added

### CODE REVIEW

File: lib/navigation/navigation.dart [hunk 1]
```diff
@@ -1 +1,10 @@
+import 'package:flutter/material.dart';
 
+class Navigation extends StatelessWidget {
+  const Navigation({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(child: Text('Navigation'),);
```
Comment (myktybayev) [inline lib/navigation/navigation.dart]: add const

File: lib/navigation/navigation.dart [hunk 2]
```diff
@@ -1 +1,10 @@
+import 'package:flutter/material.dart';
 
+class Navigation extends StatelessWidget {
```
Comment (myktybayev) [inline lib/navigation/navigation.dart]: correct class name

**Review by myktybayev — COMMENTED** [review]
fix all bugs, deadline: today

File: lib/navigation/navigation.dart (hunk 1 above)
Comment (Ulbossya) [reply lib/navigation/navigation.dart]: const can not write, cause i will get from api

**Review by myktybayev — COMMENTED** [review]
Good job!

**Pull request merged**